### ISSUE TITLE
glfwSetWindowIcon 

### ISSUE THREAD

**Issue by shaneharris** [webmixedreality/exokit#526]
I would attempt this myself but i'm not that familiar with NAN bindings ( i will be soon hopefully ) but i said i would leave here in case its just an after thought for @modulesio 

To be able to set an icon for the current window 
I assume this might drop in nicely around [here](https://github.com/webmixedreality/exokit/blob/master/deps/exokit-bindings/glfw/src/glfw.cc#L1096)

Not sure what the cleanest implementation is, do we just specify an absolute path for a PNG or ICO file. 

http://www.glfw.org/docs/latest/group__window.html#gadd7ccd39fe7a7d1f0904666ae5932dc5

**Comment by modulesio** [comment]
Yeah, the exposure of the GLWF method has a clear path that you mentioned, but I'd like to keep to webbiness for the favicon API.

That is, handling `<link rel="icon">` from the user code, and that's where we pull the favicon from. It would be fetched the same way as any other image and then passed to GLFW.

**Comment by chrisplatorres** [comment]
I'm not sure if this makes sense, typically the favicons are shown per tab in browsers, how would this be handled with multiple sites? It might make sense for exporting standalone sites packaged with Exokit.

**Comment by modulesio** [comment]
Only one site is loaded at the top level, and that site owns the icon?

**Comment by chrisplatorres** [comment]
Window title doesn't seem like the right place to renaming / changing icon dynamically. The window is "Exokit"+exokiticon or in the case of an exported site "Sites name"+icon.

**Comment by modulesio** [comment]
It probably should not be dynamic at all. Either the site sets it or we use the default.
